registerBiz.handler.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+)
+
+// BizInfo struct for Business Information from registration
+type BizInfo struct {
+	UUID      int64  `json:"uuid"`
+	Name      string `json:"name"`
+	YelpBizID string `json:"yelpBizID"`
+	Phone     string `json:"phone"`
+	Address   struct {
+		Address1 string `json:"address1"`
+		Address2 string `json:"address2"`
+		City     string `json:"city"`
+		State    string `json:"state"`
+		ZipCode  string `json:"zipCode"`
+	} `json:"address"`
+	YelpURL       string `json:"yelpURL"`
+	Img           string `json:"img"`
```

Comment:
What is this image for? are we storing an image?

Reply:
This is a dummy link to display business profile picture.